Add Bower support.

qw3rtman opened the pull request:
I created a barebones bower.json file. Additionally, I added a .gitignore for Bower. Confirm that the details in bower.json are correct.

(I have not registered the package.)

lipis commented:
### WTF?!

![screen shot 2015-05-06 at 01 39 29](https://cloud.githubusercontent.com/assets/125676/7485056/d243604c-f390-11e4-9306-d32075bc86f4.png)

qw3rtman committed:
Add bower.json.

qw3rtman committed:
Add .gitignore for Bower.

qw3rtman commented:
Seems I added the `.gitignore` for Bower after I populated `bower_components`, hence the flood of additions. :sweat_smile: 

Fixed and pushed one that is significantly lighter.

lipis commented on bower.json:
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "zoom.js",
+  "version": "0.0.1",
```
No need for version here

qw3rtman replied:
Why is that?

cvrebert replied:
Because https://github.com/bower/bower.json-spec/commit/a325da3d79baab018c572d75dc1781b12322f6cd

qw3rtman replied:
@cvrebert Ah, was not aware of that.

Removed `version`.

cvrebert commented on bower.json:
```diff
@@ -0,0 +1,26 @@
+    "Jacob Thornton <@fat>"
+  ],
+  "description": "Medium's Image Zoom for jQuery",
+  "main": "js/zoom.js",
```
Missing `css/zoom.css`

qw3rtman replied:
@cvrebert Added `css/zoom.css`.

qw3rtman committed:
Add css/zoom.css as main file in bower.json.

qw3rtman commented:
@cvrebert Added `css/zoom.css`.

qw3rtman committed:
Remove unecessary version in bower.json.

djch commented:
So this installs the whole of Bootstrap as a dependency just for transition.js?

Could this be done with git submodule or something like that?

qw3rtman commented:
I'm not familiar with submodules; however, I could register a Bower package that contains `transition.js`. Thoughts?

djch commented:
Let's ask @cvrebert, I know he's way cleverer than me.

fat commented:
i lol'd

fat commented:
@qw3rtman thank's for opening this pr. 

I feel like i'm going to reject it for now tho.

If i add 1 for bower, that sorta opens the flood gates to all these packagemangers who can't figure out their shit.

I think I'm going to bank on the gulp/grunt model.

Which is - the community can just maintain a bower-zoom if they want like they have with gulp-postcss, etc.

Just want to keep this repo pure.

That might make 0 sense. But just trying to keep things as simple as possible with this tiny project.

Thanks though!
